src/python/WMCore/Database/DBCore.py

```diff
@@ -196,3 +197,20 @@ def processData(self, sqlstmt, binds={}, conn=None,
             if not conn and connection != None:
                 connection.close() # Return connection to the pool
         return result
+
+    def convertBinds(self, bindsList, encodeType):
+        """
+        Convert any unicode string key/value to its correspondant byte string.
+        Let an exception be raised in case the string cannot be encoded in ascii.
+        :param bindsList: list of dictionaries corresponding to the binds
+        :return: updates are made in-place, but a reference is returned anyways
+        """
+        for item in bindsList:
+            for k in item:
+                ### TODO: this will have to be re-thought for python3 migration
```

Comment:
How about this instead of comment
```
import sys
....
if  sys.version.startswith('2.'): # python 2 code
```

Reply:
I added a check for the major python version, but outside of the loop. Thanks